app/src/main/java/com/mapquest/navigation/sampleapp/activity/NavigationActivity.java

```diff
@@ -746,10 +749,11 @@ private static LatLng toLatLng(Coordinate coordinate) {
         return new LatLng(coordinate.getLatitude(), coordinate.getLongitude());
     }
 
-    public static void start(Context activity, Route route) {
-        Intent intent = new Intent(activity, NavigationActivity.class);
+    public static Intent buildNavigationActivityIntent(Context context, Route route, boolean userTrackingConsentGranted) {
```

Comment:
I'm really sorry, I know we decided on passing regular intents to the NavigationNotificationService.... But perhaps it might be better to pass in PendingIntents. That way, NavigationNotificationService doesn't assume that the notification intent is an explicit Activity intent, and wrap that intent with PendingIntent.getActivity.